include/zenoh-pico/config.h.in

```diff
@@ -54,6 +54,12 @@
 #define Z_FEATURE_RX_CACHE @Z_FEATURE_RX_CACHE@
 #define Z_FEATURE_UNICAST_PEER @Z_FEATURE_UNICAST_PEER@
 #define Z_FEATURE_AUTO_RECONNECT @Z_FEATURE_AUTO_RECONNECT@
+
+/**
+ * Default session lease in milliseconds.
+ */
+#define Z_TRANSPORT_LEASE @Z_TRANSPORT_LEASE@
```

Comment:
This belongs with the other configurable parameters (lines 23-26), and I don't think the comment is needed as to be consistent with the other parameters.